### ISSUE TITLE
About loss

### ISSUE THREAD

**Issue by vincezengqiang** [meijieru/crnn.pytorch#232]
in the train.py, we calculate loss by：    cost = criterion(preds, text, preds_size, length) / batch_size
but the sample provided by pytorch does not divide batch_size, so should we divide batch_Size?

**Comment by vincezengqiang** [comment]
the default param :reduction of ctc loss is 'mean', so why still divide batchsize?
